webapp/src/test/java/com/box/l10n/mojito/service/screenshot/ScreenshotServiceTest.java

```diff
@@ -134,7 +134,7 @@ public void testDeleteScreenshotNoThirdPartyConfig() throws RepositoryNameAlread
     screenshotService.thirdPartyService = thirdPartyServiceMock;
 
     Screenshot screenshot = new Screenshot();
-    screenshot.setId(1L);
+    screenshot.setId(2L);
```

Comment:
Why is this change needed to the test?

Reply:
Same reason as above

Reply:
Before the changes, the id of the screenshot variable was updated to 29. After the changes, the id keeps as 1 (the record with that id was removed here:).
<img width="1155" alt="Screenshot 2024-07-01 at 9 08 49 PM" src="https://github.com/pinterest/mojito/assets/40587669/59a39feb-b251-43e8-8577-dc2152c6e777">